src/components/message/styles.scss

```diff
@@ -65,7 +65,7 @@
     font-size: 14px;
     line-height: 17px;
     color: theme.$color-greyscale-12;
-    display: flex;
+    display: inherit;
     flex-direction: column;
```

Comment:
1. Looks like 'block-body' contains a number of different elements. Did we verify rendering looks correct for all of the cases where other things are rendered and not just the message?
2. Can we remove the flex-direction and gap? Those only apply if you are display: flex, I think.
3. Can we just remove `display` completely here? Why does it need to inherit from the parent? I think that'd make it difficult to figure out what it's actually rendering as.

Reply:
Hey Dale!

1. I went through and checked the elements and all looked okay to me. I compared with development. From what I could see the elements render okay. Is there anything that comes to mind that would potentially be broken in the UI ? I just had another quick check and appears okay.

2. Good shout yeah I can remove those.

3. Yeah, same as above, I should be able to just remove it here. Will make that change. Thanks!

Reply:
This was partly why I didn't want to merge straight away just in case I may have missed something, felt like an update that was 'too easy' to not have caused any unwanted changes in the ui 😬

Reply:
> Is there anything that comes to mind that would potentially be broken in the UI ?

I guess mainly the gap that was there. I imagine that was probably the reason for it being flex in the first place.